SubmittedAssignments/Machhindranath_Kolape/preview.html

```diff
@@ -0,0 +1,281 @@
+            </div>
+            <div class="org-phone">
+                <!-- Bootstrap icon -->
+                <!--<i class="fas fa-whatsapp"></i>-->
+                <div class="org-phone1">
+                    <img src="img/org-whatapp.png" alt="Logo" class="org-phone1">
+                </div>
+                +91 8999417889,
+            </div>
+            <div class="org-alternate-phone">
+                <!-- Bootstrap icon -->
+                <i class="fas fa-phone"></i>
+            </div>
+            <div class="org-fax">
+                <!-- Bootstrap icon -->
+                <i class="fas fa-fax"></i>
+            </div>
+            <div class="heading-first">Learn,Explore,Grow</div>
+            <div class="heading"> AI-Driven Learning Hub!
+            </div>
+            <div class="sub-heading"><b>DIGITAL STUDY CENTER</b>
+            </div>
+            <div class="description"> <p>Welcome to <b>SwayamGuru TechLok</b>, a premier study center powered by <b>Maha
+                Mission
+                Education and Career Council</b>-an avant-garde NGO committed to sculpting bright futures. Our
+                mission
+                is to create a dynamic learning environment where individuals can thrive, explore, and grow in the
+                realm of cutting-edge technology.</p>
+            </div>
+            <!--<div class="punchline-1">
+            </div>
+            <div class="punchline-2">
+            </div>
+            <div class="offerline-1">
+            </div>
+            <div class="offerline-2">
+            </div>
+            <div class="services-heading">
+            </div>-->
+            <div class="services">
+                <div class="service">
+                    <div class="service-name">FREE MONTHLY TECHNICAL WORKSHOP</div>
+                    <div class="service-image">
+                        <!-- Bootstrap icon -->
+                        <i class="fas fa-chalkboard-teacher icon"></i>
+                    </div>
+                    <div class="service-description">Embark on a journey into <b>Future Tech:</b> Join our <b>Monthly
+                            Workshops</b> for AI Industry Updates, Hands-On Experiences, and <b>Expert Insights.</b>
+                        Stay ahead in
+                        the ever-evolving tech landscape with us!</div>
+                </div>
+            </div>
+            <div class="contact-us">
+            </div>
+            <div class="contact">CONTACT US :</div>
+            <div class="Location"><b>Location</b></div>
+            <div class="Location-image">
+                <i class="fas fa-map-marker-alt location-icon fa-lg"></i> 
+                <!--<img src="img/org-location.png" alt="Logo" class="Location-imagee">-->
+            </div>
+            <!--Bootstrap-icon-->
+            <div class="features-heading"><b><i class="fas fa-star icon"></i>FACILITIES</b></div>
+            <!--<div class="feature-imagee">
```

Comment:
do not comment the class

Reply:
I commented the classes because that not needed  in my design .